std_msgs/Header header

float32 x		# x-axis increment {-1,0,1} []
float32 y		# y-axis increment {-1,0,1} []
float32 z		# z-axis increment {-1,0,1} []
float32 yaw 	# yaw-axis increment {-1,0,1} []
